app/src/navigation/screens/VendorDetails.tsx

```diff
@@ -106,21 +126,26 @@ const VendorDetailsScreen: React.FC = () => {
           </View>
         )}
         <Button
-          onPress={() => {
+          onPress={async () => {
             if (vendor) {
               const makeUser = idToIExchange[vendor.id];
               if (!makeUser) {
                 Linking.openURL(vendor.signup);
                 return;
               }
               if (name && email) {
-                makeUser.signUp(name, email);
+                await makeUser.signUp(name, email);
+                setSigned(true);
+              } else {
+                Alert.alert(
+                  "Missing Credentials",
+                  "Please provide your name and email claims to sign up"
+                );
```

Comment:
Is this possible to get to given we disable the button?  If not, lets just remove

Reply:
depends on whether the vendor's required claims contain name & email or not
ie. vendor that doesn't required any claims would still trigger that alert